package.json

```diff
@@ -95,6 +95,7 @@
 	},
 	"devDependencies": {
 		"@apollo/react-testing": "^3.1.4",
+		"@babel/compat-data": "^7.9.0",
```

Comment:
Added this temporary, to make this failing pass:

![image](https://user-images.githubusercontent.com/1477453/79745434-f3ba2700-8310-11ea-8220-9062a456ac5a.png)

Reply:
As I wrote in another comment, this non-sense will be removed/ adjusted here: https://github.com/eventespresso/event-espresso-core/pull/2609